src/main/java/org/llorllale/cactoos/matchers/NotEmpty.java

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
@dgroup this one belongs to a different PR

Reply:
Ok, will rollback the changes.